lib/Element.js

```diff
@@ -361,12 +361,12 @@ Element.prototype.write = function (writer) {
   writer(this.name)
   for (var k in this.attrs) {
     var v = this.attrs[k]
-    if (v || (v === '') || (v === 0)) {
+    if (v != null) { // === null || undefined
       writer(' ')
       writer(k)
       writer('="')
       if (typeof v !== 'string') {
-        v = v.toString(10)
+        v = v.toString()
```

Comment:
radix 10 is the default + this may fixes issues with custom toString() methods accepting an argument